### DISCUSSION TITLE
E5108 When running neovim with lualine

### DISCUSSION THREAD

**Discussion by seestevecode** [nvim-lualine/lualine.nvim#419] [Q&A]
I am getting the following error when I try to start neovim (0.5.1) with Lualine enabled. I have tried uninstalling and reinstalling, tried with a minimal config and tried disabling all plugins but lualine.

`E5108: Error executing lua ...im/lua/lualine/components/special/function_component.lua:3: attempt to call method 'extend' (a nil value)`

I have previously had lualine working, but this has started after a recent update (I'm afraid I don't know which) and I cannot get rid of it.

Apologies if this should be an issue. I wasn't sure of the proper protocol. Thanks for any help.

**Comment by shadmansaleh** [comment]
Sorry for late reply I didn't see this.

But I can't reproduce this. Does this still occur ? Can you provide some more info ? It'll be best if you can provide a minimal reproduction config with https://github.com/nvim-lualine/lualine.nvim/blob/master/scripts/nvim_isolated_conf.sh
